query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/insert/Insert.java

```diff
@@ -44,4 +44,10 @@
    */
   @NonNull
   Insert usingTimestamp(@Nullable BindMarker bindMarker);
+
+  @NonNull
+  Insert usingTtl(int ttl);
```

Comment:
Good use of the naming conventions, we don't want to upper-case TTL here 👍

Reply:
1. Please add a `@param` tag to the javadocs and specify that the value must be understood in seconds, I think it's important to note that.
2. I would also rename the parameter to `ttlInSeconds`.

That brings the question: should we provide a method overload that takes a `Duration` instead? That could be nice, but then we would have to cope with sub-second durations, etc. Not sure if it's really helpful.

Reply:
For builders, we are extensively using `Duration` so I am +1 on that. I am not sure if we should still keep `int ttlInSeconds` method if we would provide the method that takes `Duration`

Reply:
I don't know. So far the query builder has stayed pretty close to the raw CQL syntax. If we go down that route, how about time types for the timestamp? I'd rather keep things simple rather than opening another can of worms.

Reply:
I'll revert to just an `int` and name it `ttlInSeconds`